components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,142 @@
+import { Link } from '../../i18n';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '1rem 1.5rem',
+    },
+    middleGridContainer: {
+      justifyContent: 'space-between',
+    },
+    rightGridContainer: {
+      justifyContent: 'center',
+    },
+    logo: {
+      paddingLeft: '1.5rem',
+      width: '120px',
+      height: '35px',
+    },
+    a: {
+      textDecoration: 'none',
+    },
+    link: {
+      color: theme.palette.text.primary,
+      padding: '0.5rem 1rem',
+      borderRadius: '20px',
+      fontSize: '1rem',
+      letterSpacing: '0.05rem',
+      transition: 'color 0.2s, background 0.2s',
+      '&:hover': {
+        color: 'rgb(255, 255, 255)',
+        background: theme.palette.primary.main,
+      },
+    },
+    linkBold: {
+      fontWeight: 'bold',
+    },
+  }),
+);
+
+const Navbar = (): React.ReactElement => {
+  const classes = useStyles();
+
+  return (
+    <Grid
+      container
+      direction="row"
+      justify="space-between"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item lg={2}>
+        <img
+          src="/kara_gradient.png"
+          alt="kara logo"
+          className={classes.logo}
+        />
+      </Grid>
+      <Grid
+        item
+        lg={6}
+        container
+        spacing={1}
+        className={classes.middleGridContainer}
+      >
+        <Grid item>
```

Comment:
These Grid item Link complexes should probably be made into their own little component (they get reused - with slight variation - 6 times). I know you don't share my love for components. But when you retype stuff more than twice, it's probably safe to say that code readability and maintainability would profit from moving it into its own file.